proxy/contracts/schain/DefaultAddresses.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/**
+ *   DefaultAddresses.sol - SKALE Interchain Messaging Agent
+ *   Copyright (C) 2021-Present SKALE Labs
+ *   @author Dmytro Stebaiev
+ *
+ *   SKALE IMA is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Affero General Public License as published
+ *   by the Free Software Foundation, either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   SKALE IMA is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU Affero General Public License for more details.
+ *
+ *   You should have received a copy of the GNU Affero General Public License
+ *   along with SKALE IMA.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+ pragma solidity 0.8.16;
+
+library DefaultAddresses {
+                                                          // 17 0x00 bytes
+    address public constant ETHERBASE = address(0xd2bA3e << (17 * 8));
+                                                                       // 16 0x00 bytes
+    address public constant TOKEN_MANAGER_LINKER = address(0xD2aAA008 << (16 * 8));
```

Comment:
Interesting idea to add a contract/library which have all addresses. Maybe add all addresses and use it everywhere. But have couple questions:
Would it be public or private library? I think private is much more convenient.
If it public how we would modify it (change addresses or add new addresses)?

Reply:
It's linked to contract statically and is a part of a contract itself. The idea is to have a logical module to increase code quality.
I though to add the rest on demand.

Reply:
Good I see